gnpy/core/elements.py

```diff
@@ -335,25 +302,28 @@ def carriers(self, loc, attr):
             else:
                 yield c.power._asdict().get(attr, None)
 
-    def beta2(self, ref_wavelength=1550e-9):
-        """Returns beta2 from dispersion parameter.
-        Dispersion is entered in ps/nm/km.
-        Disperion can be a numpy array or a single value.
+    def alpha(self, frequencies):
+        """ It returns the values of the series expansion of attenuation coefficient alpha(f) for all f in frequencies
+
+        :param frequencies: frequencies of series expansion [Hz]
+        :return: alpha: power attenuation coefficient for f in frequencies [Neper/m]
+        """
+        if type(self.params.loss_coef) == dict:
+            alpha_interp = interp1d(self.params.lin_loss_exp, self.params.f_loss_ref)
```

Comment:
This has been addressed - numpy.interp is now used instead of scipy.interpolate.interp1d. This method was original in fiber_params in science_utils.py and had been moved without editing.